src/main/java/org/folio/marc4ld/service/marc2ld/Marc2ldFieldRule.java

```diff
@@ -0,0 +1,50 @@
+package org.folio.marc4ld.service.marc2ld;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import org.folio.ld.dictionary.PredicateDictionary;
+import org.folio.ld.dictionary.ResourceTypeDictionary;
+import org.folio.ld.dictionary.model.Resource;
+import org.folio.marc4ld.configuration.property.Marc4BibframeRules;
+import org.marc4j.marc.ControlField;
+import org.marc4j.marc.DataField;
+
+public interface Marc2ldFieldRule {
+
+  Marc4BibframeRules.FieldRule getOriginal();
+
+  Resource computeParentIfAbsent(Resource resource);
```

Comment:
Why a "Rule" computes some 'Resource'?
"Rule" sound like a container with details how some process should go. 
But not do the processing, right?

Reply:
This comment is applicable also for methods selectResourceFromEdges, getLabel, createProperties, mergeProperties

Reply:
discussed, renamed to applier